### DISCUSSION TITLE
artalk无法正常运行

### DISCUSSION THREAD

**Discussion by tangyuanpro** [volantis-x/hexo-theme-volantis#698] [Q&A]
在volantis主题中使用artalk时，提示ReferenceError: Fancybox is not defined，无法获取评论列表数据
目前volantis版本是5.0.0-alpha.5
示例：https://yexi.pro/2021/12/26/hello-world/

**Comment by HopeLeeee** [comment]
请问你的这个问题解决了吗？我也出现了同样的问题，想请教一下。
